File: app/src/main/java/com/example/vlad/financemanager/data/models/SpinnerItem.java
package com.example.vlad.financemanager.data.models;

import java.io.Serializable;

public class SpinnerItem implements Serializable {
    public static final int POSITION_DAY = 0;
    public static final int POSITION_WEEK = 1;
    public static final int POSITION_MONTH = 2;
    public static final int POSITION_YEAR = 3;

    private String name;
    private int image;
    private int id;

    public SpinnerItem(int id, String accountName, int image) {
        this.name = accountName;
        this.image = image;
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public int getImage() {
        return image;
    }

    public int getId() {
        return id;
    }
}

File: app/src/main/java/com/example/vlad/financemanager/data/models/SpinnerItem.kt
package com.example.vlad.financemanager.data.models

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

@Parcelize
data class SpinnerItem(val id: Int, val name: String, val image: Int) : Parcelable {
    companion object {
        const val POSITION_DAY = 0
        const val POSITION_WEEK = 1
        const val POSITION_MONTH = 2
        const val POSITION_YEAR = 3
    }
}
